### PR TITLE
SONAR-6567 disable extension point NotificationDispatcher in batch

### PR DESCRIPTION
**Pull request by simonbrandhof**
SONAR-6567 disable extension point NotificationDispatcher in batch

Currently it's only removed from API. Will be removed from batch after that events are in Compute Engine (for change of quality gate status)

### CODE REVIEW

File: server/sonar-server/src/test/java/org/sonar/server/issue/notification/IssueNotificationsTest.java [hunk 1]
```diff
@@ -47,7 +47,7 @@
 //import org.sonar.api.issue.internal.DefaultIssue;
```
Comment (julienlancelot) [inline server/sonar-server/src/test/java/org/sonar/server/issue/notification/IssueNotificationsTest.java]: Is this test really usefull ?

**Comment by julienlancelot** [comment]
You need to update JavaDoc of org.sonar.api.notifications.Notification because it still referencing NotificationDispatcher.

And just for my information question, is org.sonar.api.notifications.Notification will also be removed from the API ?

**Comment by julienlancelot** [comment]
LGTM

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 new issue:
* 1 info

Details:
* Do not forget to remove this deprecated code someday. (org.codehaus.sonar:sonar-core:src/main/java/org/sonar/core/notification/NotificationDispatcher.java) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1133)

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 new issue:
* 1 info

Details:
* Do not forget to remove this deprecated code someday. (org.codehaus.sonar:sonar-core:src/main/java/org/sonar/core/notification/NotificationDispatcher.java) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1133)

**Comment by simonbrandhof** [comment]
Thanks, feedback applied.

**Comment by SonarTech** [comment]
SonarQube analysis reported 5 new issues:
* 4 major
* 1 info

Details:
* Do not forget to remove this deprecated code someday. (org.codehaus.sonar:sonar-core:src/main/java/org/sonar/core/notification/NotificationDispatcher.java) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1133)
* Add a message to this assertion. (org.codehaus.sonar:sonar-core:src/test/java/org/sonar/core/notification/NotificationChannelTest.java) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS2698)
* Add a message to this assertion. (org.codehaus.sonar:sonar-core:src/test/java/org/sonar/core/notification/NotificationChannelTest.java) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS2698)
* Add a message to this assertion. (org.codehaus.sonar:sonar-core:src/test/java/org/sonar/core/notification/NotificationDispatcherTest.java) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS2698)
* Add a message to this assertion. (org.codehaus.sonar:sonar-core:src/test/java/org/sonar/core/notification/NotificationDispatcherTest.java) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS2698)

**Pull request closed without merging**